src/providers/mbed_provider/mod.rs

```diff
@@ -7,8 +7,9 @@ use derivative::Derivative;
 use log::{error, trace};
```

Comment:
You might need to add in that file the new supported opcodes. And modify one test that is about it as well I think.